src/main/java/com/beust/jcommander/converters/DateConverter.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2010 the original author or authors.
+ * See the notice.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.beust.jcommander.converters;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Converts a String to a Date.
+ *
+ * @author Ernest Kalimullin
+ */
+public abstract class DateConverter<T> extends BaseConverter<T> {
+
+  protected static final List<String> DATE_FORMAT_LIST = new ArrayList<>();
+
+  static {
+    initPatterns(
```

Comment:
I have not read ISO 8601 in the latest edition, but I assume this list is not complete. Can you share a reference to where this list comes from?